### PR TITLE
Feat : Add Simple Profile page (logout)

### LINKED ISSUE #13: Add Profile Page (Simple)

### PR DESCRIPTION
**Pull request by ZuberDunge**
image

### CODE REVIEW

**Review by GuptaEesh — COMMENTED** [review]
The code looks clean and good to merge.

File: style/profile.css [hunk 1]
```diff
@@ -0,0 +1,82 @@
+.profile {
+    max-width: 800px;
```
Comment (AnuragSahu11) [inline style/profile.css:2]: you can use rem here

File: style/profile.css [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+    margin: 1rem auto;
+}
+
+.profile-section-grid {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    gap: 2rem;
+    margin: 2rem auto;
+    padding: .5rem;
+    max-width: 1000px;
```
Comment (AnuragSahu11) [inline style/profile.css:12]: rem

**Review by AnuragSahu11 — APPROVED** [review]
Good work, clean and formated code you can merge

File: style/profile.css (hunk 2 above)
Comment (ZuberDunge) [reply style/profile.css:12]: Thanks Anurag, I;ll keep it in mind next time

**Pull request merged**